tests/integration/network-integration.requirements.txt

```diff
@@ -1,2 +1,3 @@
 requests_toolbelt
-jsonpath-ng
\ No newline at end of file
+jsonpath-ng
+lxml
```

Comment:
You may also need to add `lxml` to the `requirements.txt` file in the root of the project.

Reply:
Done

Reply:
The other option is to make lxml optional in the same way `jsonpath-ng` is.